homeassistant/components/axis/light.py

```diff
@@ -40,6 +40,8 @@ def async_add_sensor(event_id):
 class AxisLight(AxisEventBase, LightEntity):
     """Representation of a light Axis event."""
 
+    _attr_supported_color_modes = {COLOR_MODE_BRIGHTNESS}
```

Comment:
We should not set mutable defaults as class attributes.
Instead, assign this as an instance variable in the contructor.

Reply:
Done